src/Bridge/Symfony/Form/Type/FlagBagType.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the "elao/enum" package.
+ *
+ * Copyright (C) Elao
+ *
+ * @author Elao <contact@elao.com>
+ */
+
+namespace Elao\Enum\Bridge\Symfony\Form\Type;
+
+use Elao\Enum\Bridge\Symfony\Form\DataTransformer\FlagBagToCollectionTransformer;
+use Elao\Enum\Exception\LogicException;
+use Elao\Enum\FlagBag;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Exception\InvalidConfigurationException;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @final
+ */
+class FlagBagType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        if (!$options['multiple']) {
+            throw new InvalidConfigurationException(sprintf(
+                'The "multiple" option of the "%s" form type cannot be set to false.',
+                static::class
+            ));
+        }
+
+        $builder->addModelTransformer(new FlagBagToCollectionTransformer($options['class']));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver
+            ->setDefault('multiple', true)
+            ->setAllowedValues('class', static function (string $enumType): bool {
+                if (!is_a($enumType, \BackedEnum::class, true)) {
+                    return false;
+                }
+
+                try {
+                    // this throws if some value is not power of 2
+                    FlagBag::getBitmask($enumType);
+                } catch (LogicException $e) {
```

Comment:
I think we should not even care here, the exception will be thrown with a clear enough message for the user to fix it. Let's remove this `setAllowedValues`.